Author: Agustin martin Domingo <agmartin@debian.org>
Description: Work around `define-obsolete-function-alias' different syntax for XEmacs 
Note: XEmacs `define-obsolete-function-alias' does not support optional version number
Upstream: no

XEmacs-only.
Modifies upstream:

Leo Liu (Sat Sep 28 10:54:27 2013 +0800)
    
  Rename lookup-words to ispell-lookup-words (ispell-complete-word,
  ispell-command-loop): All uses changed. Fixes: debbugs:15460

--- a/support/emacsen/ispell.el
+++ b/support/emacsen/ispell.el
@@ -2628,7 +2628,12 @@
 		(sit-for 5))
 	    (erase-buffer)))))))
 
-(define-obsolete-function-alias 'lookup-words 'ispell-lookup-words "24.4")
+;; XEmacs `define-obsolete-function-alias' does not support optional
+;; version number
+(eval-when-compile
+  (if (featurep 'xemacs)
+      (define-obsolete-function-alias 'lookup-words 'ispell-lookup-words)
+    (define-obsolete-function-alias 'lookup-words 'ispell-lookup-words "24.4")))
 
 (defun ispell-lookup-words (word &optional lookup-dict)
   "Look up WORD in optional word-list dictionary LOOKUP-DICT.
